Source/Data/Models/Device.swift

```diff
@@ -1,36 +1,74 @@
-import ObjectMapper
+import Foundation
 
-public class Device: Mappable {
-    
-    public enum PushType: Int {
+public struct Device: Codable, Equatable {
+
+    public enum PushType: Int, Codable, Equatable {
         case customWebhook = 4
         case customWebhookBehindOAuth = 5
         case fcm = 6
     }
     
+    private enum DeviceType: Int, Codable, Equatable {
```

Comment:
Mapping is from [api.parley.nu/clientApi/v1.6/doc/#operation/PostDevices.](https://api.parley.nu/clientApi/v1.6/doc/#operation/PostDevices.) This is still internal so is not accessible from outside class, but this is now more clear what the numbers do represent;